src/container/srv_target.c

```diff
@@ -278,6 +278,10 @@ cont_child_aggregate(struct ds_cont_child *cont, uint64_t *msecs)
 	if (epoch_max >= cont->sc_aggregation_max)
 		epoch_max = cont->sc_aggregation_max - 1;
 
+	if (cont->sc_ec_agg_eph_boundry > 0 &&
+	    epoch_max > cont->sc_ec_agg_eph_boundry)
+		epoch_max = cont->sc_ec_agg_eph_boundry - 1;
```

Comment:
'epoch_max' is for all objects, 'sc_ec_agg_eph_boundry' is for EC objects only.

Reply:
Yes, I will optimize this later after joe's patch.